src/base/emu-i386/simx86/trees.c

```diff
@@ -1399,9 +1400,9 @@ void InvalidateNodeRange(int al, int len)
   DoInvalidateNodeRange(al, len, NULL, NULL);
 }
 
-int TryInvalidateNodeRange(int al, int len, unsigned char *eip)
+void TryInvalidateNodeRange(int al, int len, unsigned char *eip)
```

Comment:
Maybe also the name
should be changed, as
"Try" was because it
could be interrupted.
Maybe something with
"FromFault" suffix or
alike?